docs/modules/commons/pages/expressions.adoc

```diff
@@ -40,6 +40,90 @@ Generates a random integer value between the specified origin (`minInclusive`) a
 |`1`
 |===
 
+== Math calculations
+
+=== `round`
+
+Rounds a number with an approximate value based on specific rounding definitions.
+
+[source, subs="+quotes"]
+----
+#{round(*$value*, $maxFractionDigits, $roundingMode)}
+----
+
+* *`$value`* - any number to be processed
+* `$maxFractionDigits` - maximum number of digits after the decimal separator
+* `$roundingMode` - specifies a rounding behavior for numerical operations (more information https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/math/RoundingMode.html[here]):
++
+[cols="1, 1, 4", options="header"]
+|===
+|Rouding Mode |Readable |Description
+
+|UP
+|up
+|Rounding mode to round away from zero.
```

Comment:
```suggestion
|Rounds away from zero.
```

Reply:
the same for other table rows, we shouldn't repeat information

Reply:
done